### DISCUSSION TITLE
Dark mode on pdf

### DISCUSSION THREAD

**Discussion by Cyberpunk2027GitHub** [xournalpp/xournalpp#4770] [Q&A]
Hey there!

I`ve failed to find a **dark mode** which can also be applied on a **pdf**. I`ve use **Dracula** but it didn`t help.
Any suggestion?

**Comment by danielaugot** [comment]
Same problem here. Actually, it is worse. I tried to make script  to invert a pdf to dark mode, like evince (I am always looking at PDF file in reverse mode). But when I choose to annotate such a dark PDF, xournal++ shows it white. 

My command for inverting colors (from stack overflow) is 

```
gs -o $2 -sDEVICE=pdfwrite -c "{1 exch sub}{1 exch sub}{1 exch sub}{1 exch sub} setcolortransfer" -f $1
```

**Reply by GithubAnon0000** [reply]
This command is highly dependable on your pdf viewer (see https://stackoverflow.com/a/29949191). It works for me in librewolf (129.0.1-1) and diffpdf (2.1.3-2), but not in xournalpp (1.2.3), evince (3.38.2) or libreoffice draw (24.2.5.2).
